common/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/modern-go/reflect2 v1.0.2
 	github.com/orcaman/concurrent-map v1.0.0
 	github.com/prometheus/client_golang v1.14.0
-	github.com/scroll-tech/go-ethereum v1.10.14-0.20240305121210-a807a41fd1e9
+	github.com/scroll-tech/go-ethereum v1.10.14-0.20240305174138-f128f349ba07
```

Comment:
Let's change to https://github.com/scroll-tech/go-ethereum/commit/ccec84ce63c8e768f92ad0bb5249a05e3daa0a9d

Reply:
changed in [3337b05](https://github.com/scroll-tech/scroll/pull/1178/commits/3337b055f920fb9e445f9f713e42e42ea648850b)